includes/admin.form.inc

```diff
@@ -88,6 +88,12 @@ function islandora_paged_content_admin_settings_form(array $form, array &$form_s
           'event' => 'change',
         ),
       ),
+      'islandora_paged_content_pdftotext_use_raw' => array(
+        '#type' => 'checkbox',
+        '#title' => t('Allow user to specify option of extracting raw text'),
```

Comment:
@jordandukart

Something more title like: `Allow Extraction of Raw Text` would be preferable.

Thanks!

----
William Panting
Developer
discoverygarden inc. | Managing Digital Content